kloppy/infra/serializers/tracking/epts/metadata.py

```diff
@@ -151,23 +151,42 @@ def _load_pitch_dimensions(
         return None
 
 
+def _load_provider(metadata_elm, provider: Provider) -> Provider:
+    provider_path = objectify.ObjectPath("Metadata.GlobalConfig.ProviderName")
+    provider_name = provider_path.find(metadata)
+
+    if provider:
```

Comment:
I would do the parsing just once. 
1. Read and parse provider from file into ‘provider_from_file’
2. If ‘provider’ is set, only compare with the on from file and raise warning if needed
3. Else (‘provider’ is not set), set ‘provider’ to value from file
4. Return provider

And since we only support python 3.6 please use f-strings for formatting